root/static/scripts/tests/i18n/expand2.js

```diff
@@ -131,15 +131,14 @@ test('expand2', function (t) {
   );
 
   expandHtml(
-    'A {apple_fruit|<b><strong>{prefix} APPLE!</strong></b>}',
+    'A {apple_fruit|<strong>{prefix} APPLE!</strong>}',
     {apple_fruit: 'http://www.apple.com', prefix: 'dang'},
-    'A <a href="http://www.apple.com"><b><strong>dang APPLE!</strong></b></a>',
+    'A <a href="http://www.apple.com"><strong>dang APPLE!</strong></a>',
   );
 
   expandText('{x:y|}', null, '{x:y|}');
   expandText('{x:y|}', {x: true}, 'y');
   expandText('{x:y|}', {x: false}, '');
-  expandHtml('{x:<b>|</b>|}', {x: true}, '<b>|</b>');
```

Comment:
Can you replace those with `<strong>` too? (unclear why the test was removed)

Reply:
I removed it because it used <b> but I guess it's a more general test, so sure. Will do.